### PR TITLE
[DO-2324] [Contains migration] Empty string instead null

### PR DESCRIPTION
**Pull request by Iva-khar**
Expected result:
In every "criminal_proceedings" field without data value is empty quotes

### CODE REVIEW

**Review by Andrii17s — APPROVED** [review]

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
Migrations will ends with error. You need to add **migration step** for change existing values in DB from NULL to `''`.

**Comment by Iva-khar** [comment]
> Migrations will ends with error. You need to add **migration step** for change existing values in DB from NULL to `''`.

It works locally without any errors.

**Comment by Iva-khar** [comment]
Add method for change null values to '' in migration

File: business_register/migrations/0155_auto_20210802_1752.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    model = apps.get_model('business_register', 'Pep')
+    for instance in model.objects.all():
+        if not getattr(instance, 'criminal_proceedings'):
+            setattr(instance, 'criminal_proceedings', '')
+            instance.save(update_fields=['criminal_proceedings'])
+    historical_model = apps.get_model('business_register', 'HistoricalPep')
+    for historical_instance in historical_model.objects.all():
+        if not getattr(historical_instance, 'criminal_proceedings'):
+            setattr(historical_instance, 'criminal_proceedings', '')
+            historical_instance.save(update_fields=['criminal_proceedings'])
```
Comment (roman-tiukh) [inline business_register/migrations/0155_auto_20210802_1752.py:16]:
```suggestion
def change_null_to_blank(apps, schema_editor):
    Pep = apps.get_model('business_register', 'Pep')
    for pep in Pep.objects.all():
        if not pep.criminal_proceedings:
            pep.criminal_proceedings = ''
            pep.save(update_fields=['criminal_proceedings'])
    HistoricalPep = apps.get_model('business_register', 'HistoricalPep')
    for pep in HistoricalPep.objects.all():
        if not pep.criminal_proceedings:
            pep.criminal_proceedings = ''
            pep.save(update_fields=['criminal_proceedings'])
```

**Review by roman-tiukh — APPROVED** [review]

File: business_register/migrations/0155_auto_20210802_1752.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/migrations/0155_auto_20210802_1752.py]: done

**Pull request merged**